pipeline-model-definition/src/test/resources/multipleProperties.groovy

```diff
@@ -36,7 +36,7 @@ pipeline {
     options {
         buildDiscarder(logRotator(numToKeepStr:'1'))
         disableConcurrentBuilds()
-        skipDefaultCheckout(true)
+        skipDefaultCheckout()
```

Comment:
So...what that actually means is that we should have a validation check to make sure that the `options` are unique, 'cos god only knows what the expected behavior of `properties([...])` would be. So...I'll add that check.